content/guides/developer-guides/delfin/delfin-kubernetes-guide/_index.md

```diff
@@ -0,0 +1,157 @@
+---
+title: delfin
```

Comment:
This title is used when generating the the Child menu under the main section.

![image](https://user-images.githubusercontent.com/19162717/147910135-6b29c594-cd7b-4373-89a3-cf48691f16c1.png)


Please change this to `Delfin deployment on Kubernetes`

Reply:
renamed